lists/list.css

```diff
@@ -0,0 +1,27 @@
+.list {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+ul{
+  margin:0;
```

Comment:
This should be taken care of by the CSS reset, ignore if you haven't created it. Also note that when you set an attribute for ul above, it'll apply to all uls in the DOM. Be careful about specificity rules.

Reply:
yeah... I ll keep that in mind